Raven.Database/Raft/Dto/ReplicationState.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Raven.Database.Raft.Dto
+{
+    public class ReplicationState
+    {
+        public Dictionary<string, LastModificationTimeAndTransactionalId> DatabasesToLastModification;
+
+        public ReplicationState(Dictionary<string, LastModificationTimeAndTransactionalId> databasesToLastModification)
+        {
+            DatabasesToLastModification = databasesToLastModification;
+        }
+    }
+
+    public class LastModificationTimeAndTransactionalId
+    {
+        public DateTime LastModified { get; set; }
+        public string TransactionalId { get; set; }
```

Comment:
Use `DatabaseId`, the fact that we internally use the transactional storage id is not relelvant